src/els_completion_provider.erl

```diff
@@ -67,7 +67,12 @@ find_completion( _Prefix
                , ?COMPLETION_TRIGGER_KIND_CHARACTER
                , #{trigger := <<"?">>, document := Document}
                ) ->
-  macros(Document);
+  definitions(Document, define);
```

Comment:
Completion for records and macros (and potentially other entities) works exactly in the same way, so I collapsed the logic for both into one, instead of having code duplication. We can probably extend the same strategy to other entities, too.